mcs/class/referencesource/mscorlib/system/runtime/interopservices/windowsruntime/windowsruntimemarshal.cs

```diff
@@ -1278,11 +1278,13 @@ public static IActivationFactory GetActivationFactory(Type type)
             if (type == null)
                 throw new ArgumentNullException("type");
 
+#if FEATURE_COMINTEROP || MONO_COM
```

Comment:
I had already done this change when taking my previous changes to upstream. Without it, class libs for compile for Mono.Touch and Mono.Android.